### PR TITLE
[DNM] support cypher parameter

### PR DESCRIPTION
**Pull request by czpmango**

### CODE REVIEW

File: basic_example/graph_client_basic_example.go [hunk 1]
```diff
@@ -22,16 +23,16 @@ const (
 )
 
 // Initialize logger
-var log = nebula.DefaultLogger{}
+var log = nebulago.DefaultLogger{}
```
Comment (jievince) [inline basic_example/graph_client_basic_example.go:26]: why change the package name?

Comment (czpmango) [reply basic_example/graph_client_basic_example.go:26]: Distinguish `github.com/vesoft-inc/nebula-go/v2` and `github.com/vesoft-inc/nebula-go/v2/nebula`

Comment (Aiee) [reply basic_example/graph_client_basic_example.go:26]: `nebulaCli` seems to be a more appropriate name here.

**Comment by yixinglu** [comment]
Please solve the errors and make test happy.

**Pull request closed without merging**